charts/app-config/values-production.yaml

```diff
@@ -1368,7 +1368,7 @@ govukApplications:
       cronTasks:
         - name: bigquery-export
           task: "bigquery:export"
-          schedule: "0 7 * * *"
+          schedule: "0 * * * *"
```

Comment:
Can you make this run a random minute (i.e. every 36 minute) past the hour to distribute the resource usage - as other cronjobs are likely to be set on the hr?

Reply:
sure 👍